parachain-template/runtime/Cargo.toml

```diff
@@ -69,6 +69,7 @@ cumulus-primitives-timestamp = { path = "../../primitives/timestamp", default-fe
 cumulus-primitives-utility = { path = "../../primitives/utility", default-features = false }
 pallet-collator-selection = { path = "../../pallets/collator-selection", default-features = false }
 parachain-info = { path = "../../polkadot-parachains/pallets/parachain-info", default-features = false }
+cumulus-pallet-session-benchmarking = {path = "../../pallets/session-benchmarking", default-features = false, version = "3.0.0"}
```

Comment:
why is this version 3?

Reply:
I know who we can ask :)
![image](https://user-images.githubusercontent.com/738724/141150041-482d4e35-aa7b-4b31-b35d-7cae2a2f2b57.png)

Reply:
mmh, seems that's how it was added originally and this is just copied from Statemint